text_generation/causal_lm/cpp/continuous_batching/python/python.cpp

```diff
@@ -107,4 +108,20 @@ PYBIND11_MODULE(py_continuous_batching, m) {
         .def("encode", &Tokenizer::encode)
         .def("decode", &Tokenizer::decode)
         .def("get_eos_token_id", &Tokenizer::get_eos_token_id);
+
+
+    py::class_<ov::Model, std::shared_ptr<ov::Model>> model(m, "Model");
+            model.def(py::init([](const std::shared_ptr<ov::Model>& other) {
+                return other;
+            }),
+            py::arg("other"));
+
+            model.def("get_ordered_ops", &ov::Model::get_ordered_ops,
+                R"(
+                    Return ops used in the model in topological order.
+
+                    :return: List of sorted Nodes representing ops used in model.
+                    :rtype: List[openvino.runtime.Node]
+                )");
```

Comment:
To be honest, I don't really know if I need to do this as technically ov::Model is already exposed to Python via pybind from OpenVINO, but... it didn't really work for me. I spent a lot of time trying to figure out why I can't call get_model() in Python. Once I exposed ov::Model here, it all appeared to work. Any suggestions/comments - appreciated.